### DISCUSSION TITLE
Does the exchange connector work without ticket number?

### DISCUSSION THREAD

**Discussion by lakam99** [AdhocAdam/smletsexchangeconnector#339] [Q&A] [answered]
Hello,
In my production environment, service desk management just noticed that a completed ticket continued having emails associated with the original email chain attached to the ticket. Users were not using [SRQxxxx] in the subject title. Rather, they were CC-ing the target mailbox (which is part of the 'multi-mailbox' list) and it would continue to append to the target ticket. 
This is by no means an unwanted feature, I just need confirmation if this is indeed an intentional feature or if something else is going on. Thank you!

**Comment by AdhocAdam** [comment, marked as answer]
Yes! This is an intentional feature known as "Merge Replies" that was introduced in the v1.x series of the connector.

It works by using the unique Exchange Conversation ID on the email chain to reconcile _which_ Work Item the email is about. As such, when the email is a Reply and **does not** have a Work Item in the Subject - (RE: Follow up Question) a search is performed against known conversation threads in order to identify the correct Work Item rather than create duplicate emails.

**Reply by lakam99** [reply]
Excellent feature, thanks for your response!
